integrators/embedded_explicit_runge_kutta_nyström_integrator_body.hpp

```diff
@@ -19,16 +19,18 @@ using quantities::Quotient;
 
 namespace integrators {
 
-inline EmbeddedExplicitRungeKuttaNyströmIntegrator const&
+template<typename Position>
+EmbeddedExplicitRungeKuttaNyströmIntegrator<4, 3, 4, Position> const&
 DormandElMikkawyPrince1986RKN434FM() {
-  static EmbeddedExplicitRungeKuttaNyströmIntegrator const integrator(
+  static EmbeddedExplicitRungeKuttaNyströmIntegrator<
+             4, 3, 4, Position> const integrator(
       // c
       {0.0, 1.0 / 4.0, 7.0 / 10.0, 1.0},
```

Comment:
Align the commas with those of the arrays below, it seems that they are all aligned.